pandas/io/formats/terminal.py

```diff
@@ -29,24 +27,8 @@ def get_terminal_size():
     Only to be used when running in a terminal. Note that the IPython notebook,
     IPython zmq frontends, or IDLE do not run in a terminal,
     """
-    import platform
-
-    if PY3:
-        return shutil.get_terminal_size()
-
-    current_os = platform.system()
-    tuple_xy = None
-    if current_os == 'Windows':
-        tuple_xy = _get_terminal_size_windows()
-        if tuple_xy is None:
-            tuple_xy = _get_terminal_size_tput()
-            # needed for window's python in cygwin's xterm!
-    if (current_os == 'Linux' or current_os == 'Darwin' or
-            current_os.startswith('CYGWIN')):
-        tuple_xy = _get_terminal_size_linux()
-    if tuple_xy is None:
-        tuple_xy = (80, 25)      # default value
-    return tuple_xy
+
```

Comment:
interesting, maybe can just remove the need for this function entirely?

Reply:
Yeah, that's a good point.  I'll just drop it.